lib/netext/httpext/response.go

```diff
@@ -35,7 +35,7 @@ import (
 // ResponseType is used in the request to specify how the response body should be treated
 // The conversion and validation methods are auto-generated with https://github.com/alvaroloes/enumer:
 //nolint: lll
-//go:generate enumer -type=ResponseType -transform=snake -json -text -trimprefix ResponseType -output response_type_gen.go
+//go:generate enumer -type=ResponseType -transform=title-lower -json -text -trimprefix ResponseType -output response_type_gen.go
```

Comment:
This depends on [this enumer fork](https://github.com/dmarkham/enumer) to generate `arrayBuffer` properly, so check the `*_gen.go` files changes. The older [alvaroloes/enumer](https://github.com/alvaroloes/enumer) seems unmaintained.

I included the changed `compression_type_gen.go` as well, since I ran `go generate ./lib/netext/httpext/`, but I could include just the `response_type_gen.go` changes if you prefer.